Plugins/Wox.Plugin.ControlPanel/Main.cs

```diff
@@ -74,20 +85,20 @@ public List<Result> Query(Query query)
             return panelItems;
         }
 
-        private int Score(ControlPanelItem item, string query)
+        private int Score(ControlPanelItem item, string query, bool shouldUsePinYin)
```

Comment:
Pretty sure this would not be necessary since your settings is declare in the same file, you can access it directly in the code, ie. _settings.ShouldUsePingyin ? ..........